libs/core/src/lib/flexible-column-layout/flexible-column-layout.component.html

```diff
@@ -0,0 +1,50 @@
+<div 
+    class="fd-flexible-column-layout"
+    [ngClass]="{
+        'fd-flexible-column-layout--translucent': backgroundDesign==='translucent', 
+        'fd-flexible-column-layout--transparent': backgroundDesign==='transparent'
```

Comment:
please use like this `[class.fd-flexible-column-layout--translucent]="backgroundDesign==='translucent'"`
it will be faster